README.md

```diff
@@ -1,65 +1,130 @@
 
 [![Build Status](https://travis-ci.org/fxbox/foxbox.svg?branch=master)](https://travis-ci.org/fxbox/foxbox)
 [![License](https://img.shields.io/badge/license-MPL2-blue.svg)](https://raw.githubusercontent.com/fxbox/foxbox/master/LICENSE)
 
 
-## Target
+## Technologies
 
-Target hardware for prototyping is the Raspberry Pi 2. It's ARMv7 with Neon, but building without Neon support should do for now. Target OS is the latest Raspbian which is based on Debian 8.0 Jessie.
+### Rust
 
+We're using Rust for the daemon/server. Currently v1.8.x nightly is required. As
+of 2016-02-04:
 
-## Toolchain
+```bash
+$ rustc -V
+rustc 1.8.0-nightly (dea183aa8 2016-02-02)
+```
 
-Rust 1.8+ is required. We're building with rust nightly (1.8.0 as of 2016-02-15, with ```rustc -V``` reporting: *rustc 1.8.0-nightly (fae516277 2016-02-13)*)
+It's recommended that you use [`multirust`](https://github.com/brson/multirust)
+to install and switch between versions of Rust.
 
-## Cross Compilation Toolchain
+### Node
 
-If you prefer to cross-compile on your dev system, this is the way to go. There is an extensive write-up of the process at https://github.com/japaric/rust-cross. There's also an experimental, pre-compiled toolchain at 
-https://people.mozilla.org/~fdesre/rust-rpi2.tar.gz which Fabrice built on Ubuntu 15.10. It is linked against a specific version of libstdc++.so.6 (GLIBCXX_3.4.21), so it may or may not work on other distributions. Ubuntu 15.04 is reportedly no good.
+We're using Node to run Selenium tests. Currently v4.x LTS. We plan to stay on
+stable LTS releases. It's recommended that you use
+[`nvm`](https://github.com/creationix/nvm) to install and switch between
+versions of Node.
 
- * Download and and unpack toolchain linked above in $toolchain
- * Add $toolchain/x-tools/bin and $toolchain/bin to your PATH
- * Add $toolchain/lib to your LD_LIBRARY_PATH
 
-To build a rust file:
+## Target hardware
 
-``` bash
-$ rustc --target=armv7-unknown-linux-gnueabihf -C linker=armv7-unknown-linux-gnueabihf-g++ hello.rs
-```
+We're using the Raspberry Pi 2 as a prototyping target (ARMv7). The target
+operating system is the latest Raspbian which is based on Debian 8.0 Jessie.
 
-## Building and running locally on Linux
 
-This should work straight-forward. Install a rust nightly via multirust, clone the repo, and then cargo run.
+## Contributing
 
+Note: We're in an iterative prototyping phase of the project. Things are moving
+really fast so it may be easier to contribute when the dust starts to settle.
+You've been warned. :shipit:
 
-## Building and running locally on OS X
+### Forks and feature branches
 
-``` bash
-$ brew install openssl
-$ brew install multirust
-$ multirust update
-$ multirust default nightly
+You should fork the main repo and create pull requests against feature branches
+of your fork. If you need some guidance with this see:
+
+ - https://guides.github.com/introduction/flow/
+ - http://scottchacon.com/2011/08/31/github-flow.html
+
```

Comment:
I leave two plank lines above each h2. I can remove all of these if you'd like. I think it's easier to read when looking at the plain text.